panoptes_aggregation/extractors/pluck_extractor.py

```diff
@@ -0,0 +1,35 @@
+from slugify import slugify
+from .extractor_wrapper import extractor_wrapper
+from .utilities import pluck_fields
+
+
+def slugify_or_null(s):
+    '''Slugify value while casting `null` as a string fisrt'''
+    if (s is None) or (isinstance(s, bool)) or (isinstance(s, int)):
+        return str(s)
+    else:
+        return slugify(s, separator='-')
+
+
+@extractor_wrapper()
+def pluck_extractor(classification, pluck, **kwargs):
+    '''Extract annotations from a question task into a Counter object
+
+    Parameters
+    ----------
+    classification : dict
+        A dictionary containing an `annotations` key that is a list of
+        panoptes annotations
+
+    Returns
+    -------
+    extraction : dict
+        A dictionary with the entries defined by the `pluck` argument
+
+    See also
+    --------
+    .utilities.pluck_fields
+
+    '''
+    if pluck is not None:
+        return pluck_fields(classification, pluck)
```

Comment:
Why is this its own extractor?  Wouldn't the `extractor_wrapper` also add the pluck fields resulting in them being added twice?  Might be more useful to make this a pass-through extractor that returns an empty dictionary and let the wrapper handle the plucking.

Reply:
I removed the pluck_extractor completely since it's functionality is now tied to the other extractors. Also removed the `.idea` folder and added it to `.gitignore` in case it pops up again. Also made the necessary changes to the `pluck_fields` function and added a unit tests for the question extractor. We're still looking at the other extractors and can maybe add those tests in a future PR?